components/scream/src/physics/p3/p3_constants.hpp

```diff
@@ -78,11 +79,40 @@ template <typename Scalar>
 constexpr Scalar Constants<Scalar>::QSMALL;
 
 template <typename Scalar>
+constexpr Scalar Constants<Scalar>::QTENDSMALL;
+
+template<typename Scalar>
 constexpr Scalar Constants<Scalar>::ZERO;
 
 template <typename Scalar>
 constexpr Scalar Constants<Scalar>::Tmelt;
 
+template <typename Scalar>
+constexpr Scalar Constants<Scalar>::Tol;
+
+template <typename Scalar>
```

Comment:
I removed these 2d vector globals a while ago, wondering why your PR is trying to add them back?

Reply:
Hmm, not sure.  I wonder if it was a merge issue? This PR has been around since September, I don't use these anywhere so I will remove the 2d vector globals.

Reply:
This still needs to be removed.

Reply:
I don't think you are looking at the latest commit.